cmd/tdexd/main.go

```diff
@@ -62,12 +62,13 @@ func main() {
 
 	marketsFee := int64(config.GetFloat(config.DefaultFeeKey) * 100)
 	marketsBaseAsset := config.GetString(config.BaseAssetKey)
-	tradesExpiryDuration := config.GetDuration(config.TradeExpiryTimeKey)
+	tradesExpiryDuration := config.GetDuration(config.TradeExpiryTimeKey) * time.Second
```

Comment:
afaik GetDuration returns already a time.Duration

better renames the variable in what it is so `tradesExpiryDurationInSeconds`